server/src/main/java/com/paypal/selion/grid/servlets/transfer/UploadRequestProcessor.java

```diff
@@ -295,40 +289,38 @@ private int parseRequestAsFileItems() throws FileUploadException {
         }
 
         private void populateHeadersMap() {
-            EnumMap<RequestHeaders, String> headersMap = getMapFromHeaders();
+            Map<String, String> headersMap = getRequestHeadersMap();
```

Comment:
SInce an ManagedArtifact's `instance` is created on the next line, could we change this code to use a single `instance`? 

```
ManagedArtifact instance = getManagedArtifactInstance();
Map<String, String> headersMap = getRequestHeadersMap(instance);
```
Corresponding modification to `getRequestHeadersMap()` would then be required.